ui/src/components/AppDetailedToolbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Toolbar, ToolbarGroup, ToolbarItem, Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+import './AppDetailedToolbar.css';
+
+export class AppDetailedToolbar extends React.Component {
+  onDisableApp = () => {
+    console.log('Disable button clicked');
+  }
+
+  onHomeClick = () => {
+    this.props.history.push('/');
```

Comment:
I would suggest switching this to `this.props.history.goBack();`

Reply:
using goBack() will navigate the users path rather than back through the tree of the website so that I think we should stick with it as is.